internal/rpc/headers/headers.go

```diff
@@ -24,6 +25,19 @@ const (
 	ParentRequestIDHeader = "Ftl-Parent-Request-Id"
 )
 
+var headers = []string{DirectRoutingHeader, VerbHeader, RequestIDHeader, ParentRequestIDHeader}
+
+// CopyRequest creates a new request with the same message as the original, but with only the FTL specific headers
+func CopyRequest[T any](req *connect.Request[T]) *connect.Request[T] {
```

Comment:
Small nit, this sounds like a complete copy rather than what it is? Maybe `CopyRequestForForwarding`?